src/java.base/aix/native/libnio/ch/Pollset.c

```diff
@@ -161,6 +161,23 @@ Java_sun_nio_ch_Pollset_drain1(JNIEnv *env, jclass cl, jint fd) {
     }
 }
 
+JNIEXPORT void JNICALL
+Java_sun_nio_ch_Pollset_drain(JNIEnv *env, jclass cl, jint fd)
+{
+    char buf[16];
+
+    for (;;) {
+        int n = read(fd, buf, sizeof(buf));
```

Comment:
Shouldn't we handle `errno == EINTR`? Other code uses `RESTARTABLE`.

Reply:
removed own implementation of `drain()` and use the IOUtil version instead.